frontend/src/Metamaps/Topic.js

```diff
@@ -156,7 +157,7 @@ const Topic = {
         })
       })
       if ($.isArray(nodes) && nodes.length > 1) {
```

Comment:
Are we using jquery much in this file? I wonder if we can switch to Array.isArray